website/docs/r/event_trigger.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: event_trigger"
+sidebar_current: "docs-mongodbatlas-resource-event-trigger"
+description: |-
+    Provides a Event Trigger resource.
+---
+
+# mongodbatlas_event_trigger
+
+`mongodbatlas_event_trigger` provides a Event Trigger resource. 
+
+## Example Usages
+
+### Example Usage: DATABASE with function
```

Comment:
I'd use the whole name in the title:
 ### Example Usage: Database Trigger with Function